code/modules/food_and_drinks/recipes/drinks_recipes.dm

```diff
@@ -794,4 +794,11 @@
 	name = "Red Queen"
 	id = "red_queen"
 	results = list("red_queen" = 10)
-	required_reagents = list("tea" = 6, "mercury" = 2, "blackpepper" = 1, "growthserum" = 1)
\ No newline at end of file
+	required_reagents = list("tea" = 6, "mercury" = 2, "blackpepper" = 1, "growthserum" = 1)
+	
+/datum/chemical_reaction/commander_and_chief
+	name = "Commander and Chief" 
+	id = "commander_and_chief" 
+	results = list("commander_and_chief" = 50) 
+	required_reagents = list("allies_cocktail" = 50, "champagne" = 20, "doctors_delight" = 10, "quintiple_sec" = 10, "screwdriver" = 10)
```

Comment:
```suggestion
	required_reagents = list("alliescocktail" = 50, "champagne" = 20, "doctorsdelight" = 10, "quintuple_sec" = 10, "screwdrivercocktail" = 10)
more fixies

Reply:
Heck the only one I had right was "champagne" LOL. Thanks for the help :D.

Reply:
no problemo, gotta start somewhere :heart: